index.html

```diff
@@ -1,361 +1,438 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-     <link rel="stylesheet" href="styles.css"> 
-</head>
-<body>
-  <img src="img/seven-o-clock.svg" alt="" class="logo">
-  <section class="head-line">
-    <header class="tool-bar">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home page</title>
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
- [ **OPTIONAL** ] Good job updating your page title. I suggest you improve it again to a meaningful name with a short idea about the content. As your name is **Yaseen**, you can name it like: `Yaseen's Portfolio` I think this will look nice compared to the current name.